templates/zerver/help/include/event-filtering-instruction.md

```diff
@@ -0,0 +1,12 @@
+(Optional) To filter the events that trigger the notifications, you can append
+either (or both) `&only_events=["an_event","another_event"]` or
+`&exclude_events=["an_event","another_event"]`
+with an arbitrary number of supported events to the URL.
+
+Below is a list of events that {{ integration_display_name }} bot support:
```

Comment:
If this isn't a list, then I think we need to change the phrasing here. Also since bot is singular, add an 's' to support:

> Below are the events that {{ integration_display_name }} bot supports: